index.d.ts

```diff
@@ -0,0 +1,33 @@
+declare type NodeId = string;
+declare type EdgeWeight = number;
+interface Serialized {
```

Comment:
Beautiful!

You know, this makes me think that it might still be interesting to split up this library such that only a subset of the algorithms can be imported. This can be enabled either with ES6 modules (via tree shaking) or importing from paths within the package (e.g. `import lowestCommonAncestors from 'graph-data-structure/lowestCommonAncestors'`).

These definitions might help pave the way towards that.

Related: https://github.com/datavis-tech/graph-data-structure/issues/18